src/main/java/ch/njol/skript/conditions/CondIsOnline.java

```diff
@@ -29,36 +29,43 @@
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Is Online")
-@Description("Checks whether a player is online.")
-@Examples({"player is online",
-		"player-argument is offline"})
+@Description("Checks whether a player is online. The 'connected' pattern will return false once this player leaves the server, even if they rejoin.")
+@Examples({
+	"player is online",
+	"player-argument is offline",
+	"while player is connected:",
+	"\twait 60 seconds",
+	"\tsend \"hello!\" to player"
+})
 @Since("1.4")
 public class CondIsOnline extends PropertyCondition<OfflinePlayer> {
 	
 	static {
-		register(CondIsOnline.class, "(online|1¦offline)", "offlineplayers");
+		register(CondIsOnline.class, "(online|:offline|:connected)", "offlineplayers");
```

Comment:
As I suggested on discord, this makes more sense when used even tho I would prefer it be compulsory
```suggestion
		register(CondIsOnline.class, "(online|:offline|:[still] connected)", "offlineplayers");
```

Reply:
See my earlier comment, the negated pattern will be terrible

Reply:
Ops! I missed that, my bad. I see your point there yet I am not sure about `connected` alone tho, maybe we can hear some more opinions \:)

Reply:
I don't think the negated version is bad at all. `isn't still connected` is a little awkward but perfectly legible imo.

Reply:
It's not just that it's wrong, it's also a problem that the expected correct version `is no longer connected` wouldn't be a pattern at all.

Reply:
Overall I believe not including `still` is fine but if we really do need it, then it's probably better to make this it's own condition and not an extension of `CondIsOnline`. I forgot all about the negated format of using `still` which makes it better to just have it's own condition.